guides/release/tutorial/11-route-params.md

```diff
@@ -0,0 +1,542 @@
+
+But first things first: we need to add a route for this new page. We can do that by adding a `rental` route to the router.
+
+```js { data-filename="app/router.js" data-diff="+12" }
+import EmberRouter from '@ember/routing/router';
+import config from './config/environment';
+
+export default class Router extends EmberRouter {
+  location = config.locationType;
+  rootURL = config.rootURL;
+}
+
+Router.map(function() {
+  this.route('about');
+  this.route('contact', { path: '/getting-in-touch' });
+  this.route('rental', { path: '/rentals/:rental_id' });
+});
+```
+
+Notice that we are doing something a little different here. Instead of using the default path (`/rental`), we're specifying a custom path. Not only are we using a custom path, but we're also passing in a `:rental_id`, which is what we call a _dynamic segment_. When these routes are evaluated, the `rental_id` will be substituted with the `id` of the individual rental property that we are trying to navigate to.
+
+## Links with Dynamic Segments
+
+Now that we have this route in place, we can update our `<Rental>` component to actually _link_ to each of our detailed rental properties!
+
+```js { data-filename="app/components/rental.hbs" data-diff="-7,+8,+9,+10,+11,+12" }
+<article class="rental">
+  <Rental::Image
+    src={{@rental.image}}
+    alt="A picture of {{@rental.title}}"
+  />
+  <div class="details">
+    <h3>{{@rental.title}}</h3>
+    <h3>
+      <LinkTo @route="rental" @model={{@rental}}>
+        {{@rental.title}}
+      </LinkTo>
+    </h3>
+    <div class="detail owner">
+      <span>Owner:</span> {{@rental.owner}}
+    </div>
+    <div class="detail type">
+      <span>Type:</span> {{@rental.type}}
+    </div>
+    <div class="detail location">
+      <span>Location:</span> {{@rental.city}}
+    </div>
+    <div class="detail bedrooms">
+      <span>Number of bedrooms:</span> {{@rental.bedrooms}}
+    </div>
+  </div>
+  <Map
+    @lat={{@rental.location.lat}}
+    @lng={{@rental.location.lng}}
+    @zoom="9"
+    @width="150"
+    @height="150"
+    alt="A map of {{@rental.title}}"
+  />
+</article>
+```
+
+Since we know that we're linking to the `rental` route that we just created, we also know that this route requires a dynamic segment. Thus, we need to pass in a `@model` argument so that the `<LinkTo>` component can generate the appropriate URL for that model.
+
+Let's see this in action. If we go back to our browser and refresh the page, we should see our links, and they should all link to the correct URLs.
```

Comment:
We should give a clue in this sentence that the links are incorrect. People try things out immediately and a broken state tends to surprise them. They go backwards, looking for something they did wrong.

```suggestion
Let's see this in action. If we go back to our browser and refresh the page, we should see our links, and they have URLs. But something isn't quite right yet!
```